src/sys/ioctl/platform/linux.rs

```diff
@@ -1,12 +1,27 @@
-pub const NRBITS: u32 = 8;
-pub const TYPEBITS: u32 = 8;
+/// The datatype used for the ioctl number
+#[cfg(any(target_os = "android", target_env = "musl"))]
+#[doc(hidden)]
+pub type ioctl_num_type = ::libc::c_int;
```

Comment:
I'm not certain if this is sarcastic or not, but yes, [musl defines it as an int](https://github.com/bminor/musl/blob/e738b8cbe64b6dd3ed9f47b6d4cd7eb2c422b38d/include/sys/ioctl.h#L9). Why are you surprised by that?